src/data/abilities/ability.ts

```diff
@@ -4777,17 +4777,17 @@ export class BlockCritAbAttr extends AbAttr {
   }
 
   /**
-   * Apply the block crit ability by setting the value in the provided boolean holder to false
-   * @param args - [0] is a boolean holder representing whether the attack can crit
+   * Apply the block crit ability by setting the value in the provided boolean holder to `true`.
+   * @param args `[0]` - A {@linkcode BooleanHolder} containing whether the attack is prevented from critting.
```

Comment:
It has to be `@param paramName -`, anything else is not valid.
```suggestion
   * @param args - `[0]`: A {@linkcode BooleanHolder} containing whether the attack is prevented from critting.
```